napari_svg/layer_to_xml.py

```diff
@@ -104,14 +100,11 @@ def image_to_xml(data, meta):
         if color_range != 0:
             image = image / color_range
 
-        # get colormap
-        # TODO: Currently we only support vispy colormaps, need to
-        # add support for all napari colormaps, matters for Labels too.
-        cmap = get_colormap(colormap_name)
-            
-        # apply colormap to data
-        mapped_image = cmap[image.ravel()]
-        mapped_image = mapped_image.RGBA.reshape(image.shape + (4,))
+        cmap = ensure_colormap(colormap)
+
+        # to keep backward comaptybility with napari before 0.4.18
+        # to workaround bug in `vmap.map` we need ro ravel and reshape
```

Comment:
```suggestion
        # to keep backward compatibility with napari before 0.4.18
        # because of a bug in `vmap.map`, we need to ravel, map, then reshape
```